src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/SelectPayment/_choice.html.twig

```diff
@@ -11,5 +11,6 @@
                 <p>{{ method.description }}</p>
             </div>
         {% endif %}
+        {{ sylius_template_event('sylius.shop.checkout.select_payment.choice_item_content', {'method': method}) }}
```

Comment:
Can we add the label and description to a specific template and add it as a block of this new event ?

Reply:
Agree, the way event is added now, it won't work for `mollie`, existing snippet with `method.description` needs to be extracted into external template imported back via ui event.

However, this raises a question: should it be done only for payment method choice or for shipping methods and rest of the checkout as well? cc @jakubtobiasz

Reply:
For the label, it would need to pass the form into. I don't think it could be relevant for others event, but maybe i'm wrong. 

However, I can extract descritption in that block. 

For shipping method, it could be useful too, regarding PickupPoint providers plugin for example, but maybe it should be done in another PR ?